docs/001_develop/03_client-capabilities/009_presentation/avatar.mdx

```diff
@@ -0,0 +1,212 @@
+---
+title: 'Avatar'
+sidebar_label: 'Avatar'
+id: client-presentation-avatar
+keywords: [presentation, avatar]
+tags:
+- presentation
+- avatar
```

Comment:
Can you think of any more keywords to put here? They aid searchability and it's something we've been criticised with in the path